mappings/net/minecraft/entity/EntityWithAiGoals.mapping

```diff
@@ -1,4 +1,17 @@
 CLASS us net/minecraft/entity/EntityWithAiGoals
-	FIELD a pos Ldx;
+	FIELD a aiCenter Ldx;
+	FIELD b aiRadius F
 	FIELD bv ID Ljava/util/UUID;
+	METHOD <init> (Lali;)V
+		ARG 0 world
+	METHOD a setAiRange (Ldx;I)V
+		ARG 0 center
+		ARG 1 radius
 	METHOD cO canSpawn ()Z
+	METHOD dd isInAiRange ()Z
+	METHOD de getAiCenter ()Ldx;
+	METHOD df getAiRadius ()F
+	METHOD dg setAiRangeUnlimited ()V
+	METHOD dh hasLimitedAiRange ()Z
+	METHOD f isInAiRange (Ldx;)Z
```

Comment:
`isInAiRange` → `isInHomeRadius`? See other similarly named method for other suggestions